source/table.h

```diff
@@ -40,11 +40,6 @@ typedef struct spv_operand_desc_t {
   const char* name;
   const uint32_t value;
   const libspirv::CapabilitySet capabilities;
-  // A set of extensions that enable this feature. If empty then this operand
```

Comment:
Everything but the TODO still applies.   In particular, the behaviour on empty set is not obvious without the comment.